chicago/templates/chicago/about.html

```diff
@@ -170,12 +170,14 @@ <h3 id='tags'>Legislation tags</h3>
         <p><strong>Councilmatic</strong> was originally created by <a href='https://twitter.com/mjumbewu' target="_blank">Mjumbe Poe</a> for <a href='http://philly.councilmatic.org' target="_blank">Philidelphia</a>, during his time as a 2011 <a href='http://www.codeforamerica.org/' target="_blank">Code For America</a> Fellow. It was the first fully-developed open data site for municipal legislation in the United States.</p>
 
         <p><strong>Chicago Councilmatic</strong> was <a href='http://derekeder.com/blog/keeping-tabs-on-your-local-city-council-with-councilmatic/' target="_blank">first rolled out in June 2013</a> by Derek Eder and Forest Gregg under <a href='http://opencityapps.org' target="_blank">Open City</a>.</p>
+        
+                <p><strong>Miami-Dade Councilmatic</strong> was first rolled out in June 2016</a> by <a href='http://codefor.miami' target="_blank">Code for Miami</a>.</p>
 
         <h4>Photo credits</h4>
 
         <ul class="small plain-list text-muted">
-            <li>Chicago City Hall: Chicago Ideas</li>
-            <li>Chicago City Council Chambers: Daniel X. O'Neil</li>
+            <li> </li>
```

Comment:
I actually think @wordyallen Photoshopped some of the images, but did you take the photos as well Joe?